plugins/arDominionB5Plugin/js/collapse.js

```diff
@@ -0,0 +1,17 @@
+($ => {
+
+  'use strict';
+
+  $(() => {
+    if ($('#editForm .accordion-item').length) {
+      var $collapseToShow = $(location.hash);
+      if ($collapseToShow.length) {
+        $collapseToShow.on('shown.bs.collapse', function (e) {
+          window.scrollTo(0, $(e.target).parent().offset().top);
+        });
+        bootstrap.Collapse.getOrCreateInstance($collapseToShow.get(0)).show();
```

Comment:
Nitpick: `bootstrap.Collapse.getOrCreateInstance($collapseToShow);` may be enough.

Reply:
Ah I see. I thought those methods only worked with DOM objects, but they support jQuery objects too! Address both nitpicks in https://github.com/artefactual/atom/pull/1343/commits/830719db4ddd7345613548928f2e00fbcb28a04e